### PR TITLE
[zh]Sync kubeadm-8

### PR DESCRIPTION
**Pull request by kinzhi**
related issue:https://github.com/kubernetes/website/issues/33479
kubeadm-8

```
49	36	content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_init.md
17	4	content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_upgrade_node_phase_preflight.md
16	3	content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_init_phase_certs_sa.md
18	19	content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_certs_renew_apiserver.md
```

### CODE REVIEW

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_init_phase_certs_sa.md [hunk 1]
```diff
@@ -1,3 +1,19 @@
...
+[Contributing upstream](/docs/contribute/generate-ref-docs/contribute-upstream/)
+guide. You can file document formatting bugs against the
+[reference-docs](https://github.com/kubernetes-sigs/reference-docs/) project.
+-->
+
+
+<!--
+Generate a private key for signing service account tokens along with its public key
+-->
+生成用于签署 service account 令牌的私钥及其公钥
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_init_phase_certs_sa.md:16]:
```suggestion
生成用来签署服务账号令牌的私钥及其公钥
```

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_init_phase_certs_sa.md [hunk 2]
```diff
@@ -8,20 +24,20 @@
 <!--
 Generate the private key for signing service account tokens along with its public key, and save them into sa.key and sa.pub files. If both files already exist, kubeadm skips the generation step and existing files will be used.
 -->
-
-生成用于签名 service account 令牌的私钥及其公钥，并将其保存到 sa.key 和 sa.pub 文件中。如果两个文件都已存在，则 kubeadm 会跳过生成步骤，而将使用现有文件。
+生成用于签署 service account 令牌的私钥及其公钥，并将其保存到 sa.key 和 sa.pub 文件中。如果两个文件都已存在，则 kubeadm 会跳过生成步骤，而将使用现有文件。
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_init_phase_certs_sa.md:27]:
```suggestion
生成用于签署服务账号令牌的私钥及其公钥，并将其保存到 sa.key 和 sa.pub 文件中。
如果两个文件都已存在，则 kubeadm 会跳过生成步骤，而将使用现有文件。
```

Comment (kinzhi) [reply content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_init_phase_certs_sa.md]: Has been updated

**Pull request merged**